lib/dhis2/api/deletable.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Dhis2
+  module Api
+    module Deletable
+      def delete
+        client.delete("#{self.class.resource_name}/#{id}")
+        true
```

Comment:
this looks like it can never fail - correct ? or it will raise an error if the delete call does not return a 200 ?